tests/js/spec/views/organizationGroupDetails/actions.spec.tsx

```diff
@@ -96,28 +108,52 @@ describe('GroupActions', function () {
   });
 
   describe('reprocessing', function () {
-    it('renders ReprocessAction component if org has feature flag reprocessing-v2', function () {
-      const wrapper = renderComponent();
+    it('renders ReprocessAction component if org has feature flag reprocessing-v2 and native exception event', async function () {
+      const event = TestStubs.EventStacktraceException({
+        platform: 'native',
+      });
+
+      render(
+        <GroupActions
+          group={group}
+          project={project}
+          organization={organization}
+          event={event}
+          disabled={false}
+        />
+      );
+
+      const menu = await screen.findByLabelText('More Actions');
+      userEvent.click(menu);
 
-      const reprocessActionButton = wrapper.find('ReprocessAction button');
-      expect(reprocessActionButton).toBeTruthy();
+      const reprocessActionButton = await screen.findByTestId('reprocess');
+      expect(reprocessActionButton).toBeInTheDocument();
     });
 
     it('open dialog by clicking on the ReprocessAction component', async function () {
       const event = TestStubs.EventStacktraceException({
         platform: 'native',
       });
 
-      const onReprocessEventFunc = jest.spyOn(ModalActions, 'openModal');
+      render(
+        <GroupActions
+          group={group}
+          project={project}
+          organization={organization}
+          event={event}
+          disabled={false}
+        />
+      );
 
-      const wrapper = renderComponent(event);
+      const onReprocessEventFunc = jest.spyOn(ModalActions, 'openModal');
 
-      const reprocessActionButton = wrapper.find('ReprocessAction button');
-      expect(reprocessActionButton).toBeTruthy();
+      const menu = await screen.findByLabelText('More Actions');
+      userEvent.click(menu);
 
-      reprocessActionButton.simulate('click');
+      const reprocessActionButton = await screen.findByTestId('reprocess');
+      expect(reprocessActionButton).toBeInTheDocument();
+      userEvent.click(reprocessActionButton);
       await tick();
-      wrapper.update();
 
       expect(onReprocessEventFunc).toHaveBeenCalled();
```

Comment:
if this doesn't pass without the `await tick()` try wrapping the expect with `await waitFor(() => expectxxx)`

Reply:
👍